src/types/index.ts

```diff
@@ -100,3 +100,12 @@ export interface UserRolesForDomain {
   domainId: number;
   roles: ColonyRole[];
 }
+
+export interface FixedToken {
+  chainId?: number;
+  address: Address;
+  symbol: string;
+  name: string;
+  decimals: number;
```

Comment:
Decimals are however optional. Some tokens might not have them, that's why we have the `getTokenDecimalsWithFallback` util